csrc/python_frontend/fusion_cache.cpp

```diff
@@ -118,8 +118,7 @@ FusionSchedules& FusionCache::queryFusionSchedules(size_t fusion_id) {
 }
 c10::optional<size_t> FusionCache::queryUserScheduleId(
     const FusionSchedules& scheds,
-    const at::ArrayRef<c10::IValue>& inputs,
-    int device) {
+    const at::ArrayRef<c10::IValue>& inputs) {
```

Comment:
@kevinstephano The `int device` argument was not used in the function, so I removed it.